portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DefinitionsEvents.testcase

```diff
@@ -35,12 +35,32 @@ definition {
 	}
 
 	@description = "Automation ID: LRAC-8122 | Test Summary: Show Attributes in AC"
-	@ignore = "true"
 	@priority = "5"
 	test AttributesShownInAC {
+		var attributesList = '''
+			articleId,assetId,category,className,classPK,commentId,depth,
+			elementId,entryId,fieldName,fileEntryId,fileEntryUUID,focusDuration,
+			formId,groupId,href,numberOfWords,page,pageLoadTime,preview,
+			ratingType,score,sessionId,src,tagName,text,title,type,version,viewDuration
+		''';
+
+		var list = ListUtil.newListFromString("${attributesList}");
+
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
+
+		ACUtils.launchAC();
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToDefinitions();
+
+		ACNavigation.goToDefinitionsEntity(entityName = "Event Attributes");
```

Comment:
Hi @91nilo, there's a macro that should do the same thing here: https://github.com/alexcandidos/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACDefinitions.macro#L24